src/metabase/util.clj

```diff
@@ -745,6 +745,14 @@
   [^CharSequence s]
   (.. s toString (toLowerCase (Locale/US))))
 
+(defn upper-case-en
+  "Locale-agnostic version of `clojure.string/upper-case`.
+  `clojure.string/upper-case` uses the default locale in conversions, turning
+  `ID` into `ıd`, in the Turkish locale. This function always uses the
```

Comment:
I think this docstring is backwards, this function is `upper-case` but the example is lowercasing stuff.

Anyways I think we should add a `lower-case-en` version too, just in case normal `str/lower-case` with Turkish does `ID` into `ıd